dev/conductor/core/lib/src/state.dart

```diff
@@ -28,7 +28,7 @@ const String stablePostReleaseMsg = """
   '\t\t Best practices: ${globals.hotfixDocumentationBestPractices}',
   '\t 2. Post announcement to flutter-announce group',
   '\t\t Flutter Announce: ${globals.flutterAnnounceGroup}',
-  '\t 3. Post announcement to discord',
+  '\t 3. Post announcement to discord and press the publish button.',
```

Comment:
trivial nit: you have a period on this one but not the other one (and it looks like not on the other lines?)

Reply:
Fixed!